tests/test_wave_05.py

```diff
@@ -23,13 +23,13 @@ def test_get_goals_one_saved_goal(client, one_goal):
     assert len(response_body) == 1
     assert response_body == [
         {
-            "id": 1,
+            "goal_id": 1,
```

Comment:
This is OK, but considering our specification did not use `"goal_id"` but just `"id"`, this is a break from that. It is also inconsistent with the task responses, which just use `"id"`. That could make it harder for a user of your API.